### PR TITLE
APIS-5791 Add RI email to request for production page

### PR DESCRIPTION
**Pull request by petekirby-ee**

### CODE REVIEW

**Review by robdawson-hmrc — APPROVED** [review]

File: app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/ApplicationSubmissionsPage.scala.html [hunk 1]
```diff
@@ -66,14 +66,20 @@ <h2 class="govuk-heading-m govuk-!-margin-bottom-1">@{messages("applicationsubmi
...
+          <dd id="requesterEmail" class="govuk-summary-list__value"><p class="govuk-body">@{model.currentSubmission.get.requesterEmail}</p></dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">@{messages("applicationsubmissions.requesteddate")}</dt>
+          <dd id="requestedTimestamp" class="govuk-summary-list__value"><p class="govuk-body">@{model.currentSubmission.get.timestamp}</p></dd>
+        </div>
+        @if(model.responsibleIndividualEmail.nonEmpty) {
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">@{messages("applicationsubmissions.responsibleindividualemail")}</dt>
+            <dd id="requestedTimestamp" class="govuk-summary-list__value"><p class="govuk-body">@{model.responsibleIndividualEmail}</p></dd>
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/ApplicationSubmissionsPage.scala.html:80]: Wont this output `Some(emailAddress)` to screen?

Comment (petekirby-ee) [reply app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/ApplicationSubmissionsPage.scala.html:80]: No, it outputs:

image

Comment (peteslater-ee) [reply app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/ApplicationSubmissionsPage.scala.html:80]: k

Comment (peteslater-ee) [reply app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/ApplicationSubmissionsPage.scala.html:80]: Also the id for this email element is requestedTimestamp which it probably shouldn't be

Comment (petekirby-ee) [reply app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/ApplicationSubmissionsPage.scala.html]: Fixed

**Pull request merged**